commitlint.config.ts

```diff
@@ -168,14 +168,14 @@ module.exports = {
                     ];
                 },
 
-                'reject-square-bracket-style': ({header}: {header:any}) => {
+                'type-with-square-brackets': ({header}: {header:any}) => {
```

Comment:
@realmarv but then now this rule needs to be moved down

Reply:
Done